src/interceptors/XMLHttpRequest/XMLHttpRequestOverride.ts

```diff
@@ -515,6 +516,14 @@ export const createXMLHttpRequestOverride = (
       }
     }
 
+    getResponseXML() {
+      const contentType = this.getResponseHeader('Content-Type')
+      if (contentType === 'application/xml' || contentType === 'text/xml') {
+        return new DOMParser().parseFromString(this.responseText, contentType)
```

Comment:
It appears that `DOMParser` is a browser-specific API (the name implies, but I've still double-checked):

- [MDN](https://developer.mozilla.org/en-US/docs/Web/API/DOMParser)
- [Why can't I use DOMParser in Node.js? (StackOverflow)](https://stackoverflow.com/a/55668667/2754939)

May I please ask you to verify this by adding a Node.js-oriented test, just as you've done with the JSDOM one?
You can achieve that by copying your existing test and appending the `@jest-environment node` pragma comment to make Jest run that test file in Node.js:

```js
// test/regressions/xhr-response-xml-body.node.test.ts
/**
 * @jest-environment node
 */
```

If my findings are correct, this new test would fail. Let's discuss that if that happens. Thank you!